orientdb/README.md

```diff
@@ -60,12 +60,6 @@ WARNING: Creating a new database will be done safely with multiple threads on a
 * ```orientdb.newdb``` - Overwrite the database if it already exists.
     * Only effects the ```load``` phase.
     * Default: ```false```
-* ```orientdb.intent``` - Declare an Intent to the database.
```

Comment:
@risdenk it is still present but does not have any effect and even sometimes has negative value.